longevity_test.py

```diff
@@ -116,7 +116,7 @@ def test_custom_time(self):
 
         alternator_port = self.params.get('alternator_port', default=None)
         if alternator_port:
-            endpoint_url = 'http://{}:{}'.format(self.db_cluster.nodes[0].external_address, alternator_port)
+            endpoint_url = 'http://[{}]:{}'.format(self.db_cluster.nodes[0].external_address, alternator_port)
```

Comment:
this will brake IPv4

Reply:
no it won't break it, we tested it

Reply:
We ran such http command with ipv4 - it worked